plugins/editor/src/editor/Editor.cpp

```diff
@@ -311,6 +324,7 @@ void Editor::Impl::uiReceiveValue(EditId id, const EditValue& v)
             sendQueuedOSC("/key/slots", "", nullptr);
             sendQueuedOSC("/sw/last/slots", "", nullptr);
             sendQueuedOSC("/cc/slots", "", nullptr);
+            sendQueuedOSC("/image", "", nullptr);
```

Comment:
To add also in the other places of the same file, which send this same OSC sequence